frontend/packages/pipelines-plugin/src/components/pipelineruns/detail-page-tabs/PipelineRunCustomDetails.tsx

```diff
@@ -59,6 +65,37 @@ const PipelineRunCustomDetails: React.FC<PipelineRunCustomDetailsProps> = ({ pip
           <PipelineResourceRef {...convertBackingPipelineToPipelineResourceRefProps(pipelineRun)} />
         </dd>
       </dl>
+      {buildImage && (
+        <dl>
+          <dt>{t('pipelines-plugin~Download SBOM')}</dt>
+          <dd>
+            <ClipboardCopy
+              isReadOnly
+              hoverTip={t('pipelines-plugin~Copy')}
+              clickTip={t('pipelines-plugin~Copied')}
+            >
+              {`cosign download sbom ${buildImage}`}
+            </ClipboardCopy>
+            <ExternalLink href="https://docs.sigstore.dev/cosign/installation">
+              {t('pipelines-plugin~Install Cosign')}
+            </ExternalLink>
+          </dd>
+        </dl>
+      )}
+      {sbomTaskRun && (
+        <dl>
+          <dt>{t('pipelines-plugin~SBOM')}</dt>
+          <dd>
+            <Link
+              to={`/k8s/ns/${sbomTaskRun.metadata.namespace}/${referenceForModel(TaskRunModel)}/${
+                sbomTaskRun.metadata.name
+              }/logs`}
+            >
+              {t('pipelines-plugin~View SBOM')}
+            </Link>
```

Comment:
I don't know your designs, but a `ResourceLink` might be a good option here. Wdyt?

Reply:
@jerolimov This is a taskruns logs link, I think ResourceLink is mainly used for redirecting to resource details page